src/analysis/special_functions/gamma.lean

```diff
@@ -7,6 +7,7 @@ import measure_theory.integral.exp_decay
 import analysis.calculus.parametric_integral
 import analysis.special_functions.integrals
 import analysis.convolution
+import analysis.special_functions.trigonometric.euler_sine_prod
 
 /-!
 # The Gamma and Beta functions
```

Comment:
Could you update the file-level docstring to mention the new results (notably, the reflection formula and the non-vanishing)?